sdk/ai/azure-ai-agents/samples/agents_multiagent/sample_agents_agent_team.py

```diff
@@ -7,16 +7,30 @@
 DESCRIPTION:
     This sample demonstrates how to use multiple agents using AgentTeam with traces.
 
+    The team consists of
+        • one leader agent - automatically created by AgentTeam from the
+          configuration in `utils/agent_team_config.yaml`
+        • two worker agents - `Coder` and `Reviewer`, defined in the code below
+
+    IMPORTANT - leader-agent model configuration
+        `utils/agent_team_config.yaml` contains the key  TEAM_LEADER_MODEL.
+        Its value must be the name of a **deployed** model in your Azure AI
+        project (e.g. "gpt-4o-mini").  
+        If this model deployment is not available, AgentTeam cannot instantiate
+        the leader agent and the sample will fail.
+
 USAGE:
     python sample_agents_agent_team.py
 
     Before running the sample:
 
-    pip install azure-ai-agents azure-identity
-
-    Set these environment variables with your own values:
-    PROJECT_ENDPOINT - the Azure AI Agents endpoint.
-    MODEL_DEPLOYMENT_NAME - the name of the model deployment to use.
+    1. pip install azure-ai-agents azure-identity
+    2. Ensure `utils/agent_team_config.yaml` is present and TEAM_LEADER_MODEL points
+       to a valid model deployment.
+    3. Set these environment variables with your own values:
+         PROJECT_ENDPOINT - the Azure AI Agents endpoint.
```

Comment:
"PROJECT_ENDPOINT - The Azure AI Project endpoint, as found in the Overview page of your Azure AI Foundry portal."

Reply:
Thanks, updated